pageserver/src/tenant.rs

```diff
@@ -1858,6 +1858,7 @@ impl Tenant {
                 });
             })
         };
+        // test_long_timeline_create_then_tenant_delete is leaning on this message
         tracing::info!("Waiting for timelines...");
```

Comment:
I think this is something which was forgotten to demote to debug in some earlier PR. I'd rather get rid of it later.

Reply:
Instead the deletion should log ... or something. There is no logging in the prepare path, and that is more rare than shutting down.